ix-dev/community/esphome/README.md

```diff
@@ -0,0 +1,3 @@
+# esphome
+
+[esphome](https://github.com/esphome/esphome) is a PVR for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new episodes of your favorite shows and will grab, sort and rename them.
```

Comment:
That's not the description for esphome.

Reply:
fixed in [182bc346](https://github.com/truenas/apps/pull/1406/commits/182bc34606d189dc253d89a4a27be5dfe78ba827)